pkg/cmd/adm/install_operator.go

```diff
@@ -43,17 +44,18 @@ func NewInstallOperatorCmd() *cobra.Command {
 
 			cl := commonclient.NewApplyClient(kubeClient)
 			ctx := clicontext.NewTerminalContext(term)
-			return installOperator(ctx, commandArgs, args[0], time.Second*60, cl)
+			return installOperator(ctx, commandArgs, args[0], cl)
 		},
 	}
 
 	cmd.Flags().StringVar(&commandArgs.kubeConfig, "kubeconfig", "", "Path to the kubeconfig file to use.")
 	flags.MustMarkRequired(cmd, "kubeconfig")
 	cmd.Flags().StringVar(&commandArgs.namespace, "namespace", "", "The namespace where the operator will be installed. Host and Member should be installed in separate namespaces. If the namespace is not provided the standard namespace names are used: toolchain-host|member-operator.")
+	cmd.Flags().DurationVar(&commandArgs.waitForReadyTimeout, "timeout", time.Second*180, "The max timeout used when waiting for each of the resources to be installed.")
```

Comment:
```suggestion
	cmd.Flags().DurationVar(&commandArgs.waitForReadyTimeout, "timeout", time.Second*180, "The max timeout used when waiting for each of the resources to be installed. The default timeout, if not specified, is 3m.")
```

Reply:
Thanks, I can add it, but it's already showed by cobra args:
```
✗ go run cmd/ksctl/main.go adm install-operator
Error: required flag(s) "kubeconfig" not set
Usage:
  ksctl adm install-operator <host|member> --kubeconfig <path/to/kubeconfig> --namespace <namespace> [flags]

Flags:
  -h, --help                help for install-operator
      --kubeconfig string   Path to the kubeconfig file to use.
      --namespace string    The namespace where the operator will be installed. Host and Member should be installed in separate namespaces. If the namespace is not provided the standard namespace names are used: toolchain-host|member-operator.
      --timeout duration    The max timeout used when waiting for each of the resources to be installed. (default 3m0s)

Global Flags:
  -y, --assume-yes      Automatically answer yes for all questions.
      --config string   config file (default is $HOME/.ksctl.yaml)
  -v, --verbose         print extra info/debug messages

required flag(s) "kubeconfig" not set
```

Reply:
Ah, ok. Than't it fine to keep it as is.